test/metabase/db/setup_test.clj

```diff
@@ -36,3 +43,38 @@
         (testing "from a connection URL"
           (test* (mdb.data-source/raw-connection-string->DataSource
                   (str "jdbc:h2:" subname))))))))
+
+(deftest setup-fresh-db-test
+  (mt/test-drivers #{:h2 :mysql :postgres}
+    (testing "can setup a fresh db"
+      (mt/with-temp-empty-app-db [_conn driver/*driver*]
+        (is (= :done
+               (mdb.setup/setup-db! driver/*driver* (mdb.connection/data-source) true)))))))
```

Comment:
What about checking if the expected changes are (or the last expected change is) there?